internal/controller/controller.go

```diff
@@ -53,10 +53,21 @@ func (r *Route) Reconcile(ctx context.Context, req reconcile.Request) (reconcile
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+	if len(route.ObjectMeta.OwnerReferences) > 0 {
+		for _, o := range route.ObjectMeta.OwnerReferences {
+			if o.Kind == "Ingress" {
+				log.V(5).Info("ignoring route owned by ingress", o.Name)
```

Comment:
**Suggestion(blocking):** I suggest to add a test case that checks that when a Route is owned by an Ingress, the Route won't be reconciled by openshift-routes.

Reply:
I have added a quick test for that in 6f0eddb221653ea7f6cec59de1841d11ddebf28c. I should probably have done much better, but the way everything revolves around the `Route` struct, I figured I'd go with a simple test like this.